test/EnvironmentManagerTests/packages.config

```diff
@@ -1,6 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.Taef" version="10.58.210222006-develop" targetFramework="native" />
+  <package id="Microsoft.VCRTForwarders.140" version="1.0.7" targetFramework="native" />
```

Comment:
Why?

We're doing the Hybrid CRT thing. See https://github.com/microsoft/WindowsAppSDK/pull/1026 for more info

Reply:
Because, when I made this I needed the VCRTForwaders to get my Win32 tests to work.  RegFreeWinrt.  What do I need to grab from that PR so I can get both my Win32 and Centennial tests running?